src/components/highlight/CommentComponent.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Modal } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import '../../styles/style.css';
+
+const HighlightComment = ({ showModal, handleModalClose, section, article,
+  comment, getComment, handleSubmit, baseOffset, extentOffset,
+  containsHighlight, updateComment, currentUser }) => {
+  const { author } = article;
+  const commentedSection = containsHighlight(article, section, baseOffset, extentOffset);
```

Comment:
Would you put the properties in a new line

Reply:
I've implemented that, thanks